### PR TITLE
Spells/DK: Fix talent permafrost

### PR DESCRIPTION
**Pull request by aquadeus**
**Changes proposed:**

-  Fix talent permafrost

**Tests performed:**

Tested in-game

### CODE REVIEW

**Comment by mdX7** [comment]
missing validate for `SPELL_DK_FROST_SHIELD`

**Comment by aquadeus** [comment]
> missing validate for `SPELL_DK_FROST_SHIELD`

mb

File: src/server/scripts/Spells/spell_dk.cpp [hunk 1]
```diff
@@ -710,6 +711,29 @@ class spell_dk_obliteration : public AuraScript
...
+        GetTarget()->CastSpell(GetTarget(), SPELL_DK_FROST_SHIELD, CastSpellExtraArgs(aurEff).AddSpellMod(SPELLVALUE_BASE_POINT0, _damageAmount));
+    }
+
+    void Register() override
+    {
+        OnEffectProc += AuraEffectProcFn(spell_dk_permafrost::HandleEffectProc, EFFECT_0, SPELL_AURA_DUMMY);
+    }
+
+private:
+    uint32 _damageAmount;
```
Comment (Shauren) [inline src/server/scripts/Spells/spell_dk.cpp:734]: Why is this a class member at all? Remove that

**Review by Shauren — CHANGES_REQUESTED** [review]

File: src/server/scripts/Spells/spell_dk.cpp (hunk 1 above)
Comment (aquadeus) [reply src/server/scripts/Spells/spell_dk.cpp]: done

**Pull request merged**